packages/runtime-handler/src/dev-runtime/route.ts

```diff
@@ -303,25 +303,46 @@ export function functionPathToRoute(
         reply,
         debugMessage,
         debugArgs = [],
+        crossForkLogMessage,
       }: {
         err?: Error | number | string;
         reply?: Reply;
         debugMessage?: string;
         debugArgs?: any[];
+        crossForkLogMessage?: {
+          level: keyof LoggerInstance;
+          args: [string] | [string, number] | [string, string];
```

Comment:
I can't imagine these ever changing, but does it make sense to colocation the potential args shapes with the LoggerInstance type? In case the logger instance type is used similarly elsewhere.

Reply:
args in the sendLog has the type `(string | number)[]` . This was cleaner to read.